src/seedu/addressbook/AddressBook.java

```diff
@@ -257,16 +273,16 @@ private static void echoUserCommand(String userCommand) {
      * @param args full program arguments passed to application main method
      */
     private static void processProgramArgs(String[] args) {
-        if (args.length >= 2) {
+        if (args.length >= FILE_MAX_LIMIT) {
```

Comment:
constants name don't seem to match their purpose, but good that you found magic constants.